libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -303,6 +305,16 @@ void AP_MotorsMulticopter::update_throttle_filter()
     } else {
         _throttle_filter.reset(0.0f);
     }
+
+    float new_thr = _throttle_filter.get();
+
+    if (!is_equal(last_thr, new_thr)) {
+        _throttle_slew.update(new_thr, AP_HAL::micros());
```

Comment:
I would have thought you want to slew the throttle output from motors not the input? Input does not take into account throttle mix boosting throttle.

Reply:
betaflight slews from the input - this is essentially a FF term so wants to be on the earliest predicated change

Reply:
That would risk some additional feedback loops. Given this is still developmental I think the current approach is safer.